backendRoot/build.gradle

```diff
@@ -2,6 +2,7 @@ dependencies {
   junit()
   powerMock()
   slf4j()
+  slf4jApi()
```

Comment:
I don't think you need both, just `slf4j()` should be enough.

`slf4j-simple`, just as any other slf4j implementation, pulls in `slf4j-api` transitively anyway (see https://mvnrepository.com/artifact/org.slf4j/slf4j-simple/1.7.28)

Reply:
Whoops, not true, disregard the comment - yes you need both indeed; the naming is currently confusing, I forgot that `slf4j()` applies in test context only.

Reply:
Hmm likely to make it completely right you'll also need to actually exclude transitive dependency from `slf4j-simple` to `slf4j-api`... otherwise you might end up with two different versions of `slf4j-api` in test classpath.